### ISSUE TITLE
Bug - ClipboardCopy - some examples are missing aria-expanded attribute

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly#7271]
**Describe the problem**
This is an issue with 
- [ ] Patternfly 5  
- [x] Patternfly 6  
- [ ] other

Some examples are missing the `aria-expanded` attribute when the clipboard copy is not expanded. For expansion, `aria-expanded` must always be present with either a true or false value; omitting the attribute provides a different context than passing a false value.

**How do you reproduce the problem?**
Go to the [expandable clipboard copy examples](https://www.patternfly.org/components/clipboard-copy/html#expandable) and inspect the DOM; notice that the examples that are not expanded are missing `aria-expanded="false"`.

**Expected behavior**
`aria-expanded` is always present when expansion is possible.

**Is this issue blocking you?**
List the workaround if there is one.

**Screenshots**
If applicable, add screenshots to help explain the issue.

**What is your environment?**

**What is your product and what release date are you targeting?**

**Any other information?**
